packages/devtools/client/package.json

```diff
@@ -71,6 +73,8 @@
     "type-fest": "^4.1.0",
     "typescript": "~5.0.4",
     "ufo": "^1.3.0",
-    "valtio": "^1.11.1"
+    "valtio": "^1.11.1",
+    "babel-jest": "^29.5.0",
```

Comment:
why not @swc/jest

Reply:
can we just extends @scripts/jest-config

Reply:
I forgot about the shared configs. Let me rewrite it.

Reply:
Done~